AmazonChimeSDK/AmazonChimeSDK/internal/video/DefaultVideoClientController.swift

```diff
@@ -19,7 +19,7 @@ class DefaultVideoClientController: NSObject {
     var videoClientState: VideoClientState = .uninitialized
     let videoTileControllerObservers = ConcurrentMutableSet()
     let videoObservers = ConcurrentMutableSet()
-    var dataMessageObservers = [String: ConcurrentMutableSet]()
+    var dataMessageObservers = ConcurrentDictionary<String, NSMutableSet>()
```

Comment:
What is this change for?

Reply:
This fixes a race condition that was causing a crash when ConcurrentMutableSet was released too early.